Podfile

```diff
@@ -3,8 +3,8 @@ platform :osx, '10.12'
 target 'alt-tab-macos' do
   use_frameworks!
   pod 'LetsMove', :git => 'https://github.com/lwouis/LetsMove.git', :commit => '7abf4daed1a25218f2b52f2dfd190aee5a50071c'
-  pod 'Sparkle', :podspec => 'https://raw.githubusercontent.com/lwouis/Sparkle/master/Sparkle.podspec'
+  pod 'Sparkle', '1.24.0'
```

Comment:
The reason I was pointing to a fork of Sparkle, is because in that fork, I addressed an issue that Sparkle has, that impacts release notes in AltTab. [See the 3 commits I added in that fork](https://github.com/lwouis/Sparkle/commits/master). Moving back to upstream may create a regression here

Reply:
Actually I checked Sparkle 1.24.0 release notes, and they have merged my fix (I had shared a PR with them back then). The issue is that since then I've updated my fix further. I should share my improvements in a PR, and re-fork Sparkle in the meanwhile.